aspnetcore/host-and-deploy/aspnet-core-module.md

```diff
@@ -492,6 +492,30 @@ The ASP.NET Core Module installer runs with the privileges of the **SYSTEM** acc
 1. Export the updated *applicationHost.config* file to the share.
 1. Re-enable the IIS Shared Configuration.
 
+::: moniker range=">= aspnetcore-2.2"
+
+## Application Initialization
+
+App initialization is achieved for apps that use the [in-process hosting model](xref:fundamentals/servers/index#in-process-hosting-model) with [IIS Application Initialization](/iis/get-started/whats-new-in-iis-8/iis-80-application-initialization):
```

Comment:
```suggestion
App initialization is an IIS feature that sends a fake HTTP request to the app. It can be used by both the [in-process hosting model](xref:fundamentals/servers/index#in-process-hosting-model) and [out-of-process hosting model](xref:fundamentals/servers/index#out-of-process-hosting-model) with [IIS Application Initialization](/iis/get-started/whats-new-in-iis-8/iis-80-application-initialization):
```

General suggestion. Make it grammatically fit here.

Reply:
It doesn't work out-of-process tho.

Reply:
I'll double-check, but I thought both Sourabh said so and my testing had the same outcome. I'll check again to make sure.

Reply:
It should work with out of process. We have tests for it: https://github.com/aspnet/AspNetCore/blob/35746adf68798336bbb6d0f709b66a394c2734e9/src/Servers/IIS/IIS/test/IIS.Shared.FunctionalTests/ServicesTests.cs#L33

Reply:
Check between AspNetCoreModule and AspNetCoreModuleV2

Reply:
oh ok ... sounds good ... must have been ANCMv1 that I'm thinking of (hence all of the earlier requests about the feature).

Reply:
btw - I found Sourabh's remark ...

https://github.com/aspnet/AspNetCore/issues/3849#issuecomment-432753492

... implies in-proc only. No worries ... I'll test it both ways.

Reply:
Testing confirms that both `StartMode` = `AlwaysRunning` and `Preload Enabled` = `True` must be set. Setting either of them doesn't work.

Testing out-of-proc with ANCMv2: :+1: :tada:

Testing out-of-proc with ANCMv1: :-1: :cry: ... but expected.